app/gateway/2.6.x/install-and-run/debian.md

```diff
@@ -0,0 +1,62 @@
+---
+id: page-install-method
+title: Install Kong Gateway on Debian
+header_title: Debian Installation
+header_icon: /assets/images/icons/icn-installation.svg
+breadcrumbs:
+  Installation: /install
+---
+
+## Prerequisites
+
+You have a supported system with root or [root-equivalent](/gateway/{{page.kong_version}}/plan-and-deploy/kong-user) access.
+
+## Install
+
+You can install Gateway by downloading an installation package or using our `apt` repository.
+
+{% navtabs %}
+{% navtab Install with packages %}
+
+    {% if include.distribution == "ubuntu" %}
+
+- [Xenial]({{ site.links.download }}/gateway-2.x-ubuntu-xenial/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+- [Bionic]({{ site.links.download }}/gateway-2.x-ubuntu-bionic/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+- [Focal]({{ site.links.download }}/gateway-2.x-ubuntu-focal/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+
+    {% endif %}
+
+    {% if include.distribution == "debian" %}
+
+- [8 Jessie]({{ site.links.download }}/gateway-2.x-debian-jessie/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+- [9 Stretch]({{ site.links.download }}/gateway-2.x-debian-stretch/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+- [10 Buster]({{ site.links.download }}/gateway-2.x-debian-buster/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+- [11 Bullseye]({{ site.links.download }}/gateway-2.x-debian-bullseye/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
+
+    {% endif %}
```

Comment:
```suggestion
```
Moved all of this into the banner at the top, and removed Ubuntu links, since those got moved to the Ubuntu topic.